test/spark/app/.gitignore

```diff
@@ -0,0 +1,34 @@
+*.class
+*.log
+*.zip
+*.gz
+*.jpeg
+*.jpg
+*.png
+*~
```

Comment:
Definitely *don't* `.gitignore` this here, this is editor-specific configuration.  Emacs fankidz can put this in their ` ~/.config/git/ignore` file, right next to a line `#*` that's anyway also potentially needed.

Reply:
will reduce the new gitignore number of entries